### DISCUSSION TITLE
Fresh Install - Schema graphql does not exist

### DISCUSSION THREAD

**Discussion by jeffwood13** [twentyhq/twenty#2408] [Ideas]
After debugging and completing a second fresh Local Installation, I am receiving the following error(s):

useFindManyObjects for "viewsV2" error : ApolloError: schema "graphql" does not exist

The error is repeatable for any action that uses the graphql schema.

I checked the code and did not see where the schema would be seeded to the database, should it have been?

Any guidance or direction would be appreciated.

Thank you.

**Comment by charlesBochet** [comment]
Hi @jeffwood13, feel free to join our discord here https://discord.gg/cx5n4Jzs57 to get help. We are more reactive there

Regarding your issue, it seems that pg_graphql is not correctly installed.
Depending on your database setup, you need to run the setup script:
- postgres on Linux (or WSL): `./infra/dev/scripts/setup-postgres-linux.sh`
- postgres on Mac: `./infra/dev/scripts/setup-postgres-macos.sh`
- postgres in Docker: `make provision-postgres`

You'll find more information here: https://docs.twenty.com/contributor/local-setup/yarn-setup

**Comment by jeffwood13** [comment]
After a clean installation and specifying version 14 when installing postgres, this issue has been resolved.
